src/components/App/App.js

```diff
@@ -11,15 +11,33 @@ import familyTree from "../../assets/family-tree.png";
 import galleryImg from "../../assets/gallery.png"
 import { CountdownPane } from "../CountdownPane/CountdownPane";
 import { ExecutorPane } from "../ExecutorPane/ExecutorPane";
+import PhotoAdd from "../PhotoAdd/PhotoAdd";
 import ObitPane from "../ObitPane/ObitPane";
 import { Switch, NavLink, Link, Route } from 'react-router-dom';
 
 const App = () => {
-  const [user, setUser] = useState({})
+  const [user, setUser] = useState({
+      "id": "",
+      "type": "",
+      "attributes": {
+          "email": "",
+          "name": "",
+          "date_of_birth": "",
+          "obituary": "",
+          "profile_picture": "",
```

Comment:
You should change this to be the correct name that will be coming back from the fetch.

Reply:
Removed this!